oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/Metrics.java

```diff
@@ -56,8 +56,9 @@
      * Merge the given metrics instance, these two must be the same metrics type.
      *
      * @param metrics to be merged
+     * @return {@code true} if the combined metrics should be continuously processed, {@code false} it should be abandoned.
```

Comment:
```suggestion
     * @return {@code true} if the combined metrics should be continuously processed. {@code false} means it should be abandoned, and keep the data unaltered in this case.
```
I add a little more to the comments, from my code reading, if the change has been made, the data will still flush into the database eventually.

Reply:
Also, I feel the data in the cache has refreshed repeatedly into database even without change, could you recheck about this too? If so, we need to fix that in another PR, which should be able to reduce the database load.

Reply:
> Also, I feel the data in the cache has refreshed repeatedly into database even without change, could you recheck about this too? If so, we need to fix that in another PR, which should be able to reduce the database load.

@kezhenxu94 ⬆️ What do you think about this?

Reply:
Because only limited `Event` data falls into this case, this may be not a big problem for now. Anyway, let me check and think about how to deal with this

Reply:
I don't mean the change you made, I mean for all metrics, if the metrics exist in the cache, even no update, they are keeping being updated periodly.

Reply:
> I don't mean the change you made, I mean for all metrics, if the metrics exist in the cache, even no update, they are keeping being updated periodly.

OK I mistakenly thought you only referred to the "abandoned" metrics, it makes sense to me now, I'll check